upgrade_compatibility.md

```diff
@@ -0,0 +1,12 @@
+# Upgrade Compatibility
+
+While we try and make all changes backwards-compatible, sometimes we have to make changes that mean you cannot cleanly provision a new environment.
+
+
+## DNS resources already exist
+
+Error:
+
+`Error creating DNS RecordSet: googleapi: Error 409: The resource 'entity.change.additions[0]' named 'env-proxy.tsuru2.paas.alphagov.co.uk. (A)' already exists, alreadyExists`
+
+This is caused by `7780fe9d`, which also contains the scrips you can use to fix it.
```

Comment:
Interesting that this only works in master. I have viewed this not in code, but normally via github web, just selecting branch (because I wanted to click the links to see how it works). I see no reason this not working, as that commit is present even on this branch... Otherwise all fine.